packages/generator-foodq/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "generator-foodq",
   "version": "1.10.5",
   "private": true,
-  "description": "A sample generator to preview all the options available when using the Yeoman-UI",
+  "description": "A sample generator to preview all the options available when using the Yeoman-UI (updated)",
```

Comment:
I didn't get why need to update the description?

Reply:
Thats a mistake I can revert